foundation-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -2101,7 +2199,46 @@ Identity.syncIdentifier("idType",
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
+#### iOS
+
+**Syntax**
+
+```swift
+@objc(syncIdentifierWithType:identifier:authenticationState:)
+static func syncIdentifier(identifierType: String, identifier: String, authenticationState: MobileVisitorAuthenticationState)
+```
+
+* The _identifierType \(String\)_ contains the `identifier type`, and this parameter should not be null or empty.
+* The _identifier \(String\)_ contains the `identifier` value, and this parameter should not be null or empty.
+
+  If either the `identifierType` or `identifier` contains a null or an empty string, the identifier is ignored by the Identity extension.
+
+* The _authenticationState \(MobileVisitorAuthenticationState\)_ value indicates the authentication state for the user and contains one of the following `MobileVisitorAuthenticationState` values:
+  * `MobileVisitorAuthenticationState.authenticated`
+  * `MobileVisitorAuthenticationState.loggedOut`
+  * `MobileVisitorAuthenticationState.unknown`
```

Comment:
For these sections we should use a link to public classes section instead of redefining them.